blocks/identity-block/components/Identity.jsx

```diff
@@ -0,0 +1,26 @@
+// import { useState } from 'react';
+import Identity from '@arc-publishing/sdk-identity';
+// import { useFusionContext } from 'fusion:context';
+// import getProperties from 'fusion:properties';
+// import { isServerSide } from '@wpmedia/engine-theme-sdk';
+
+const useIdentity = () => ({
+  Identity,
+  isInitialized: true,
+});
+
+// keep skelton for future use
```

Comment:
I've no issue with uncommenting this, if you're ok with it

Reply:
will uncomment when the next ticket comes in -- good idea